client/src/components/CRM/Accounts/ManageUsers/ManageAccounts.tsx

```diff
@@ -88,12 +88,21 @@ export default function ManageAccounts() {
   };
   const submitNewUser = async (e: any) => {
     e.preventDefault();
+    /** @param {string} token - AccessToken */
+    const token = await getAccessTokenSilently({
+      audience: 'https://localhost:8000',
+      scope: 'admin',
+    });
     /** @param {Response} r - Post data to Accounts Table */
     const r = await fetch(process.env.REACT_APP_API_1_URL + `/accounts`, {
```

Comment:
This was probably already part of the code but I don't think using JSDoc for API calls is necessary. Renaming the const r variable to be more descriptive to what the data returns represents is self-documenting enough.

Reply:
I removed the JSDoc comments for these cases, renamed the variables and also reformatted the JSX while I was at it